docs/getting-started-guides/coreos/azure/lib/azure_wrapper.js

```diff
@@ -107,6 +107,9 @@ var create_ssh_key = function (prefix) {
   };
   openssl.exec('req', opts, function (err, buffer) {
     if (err) console.log(clr.red(err));
+    openssl.exec('rsa', { in: opts.keyout, out: opts.keyout }, function (err, buffer) {
+       if (err) console.log(clr.red(err));
+    });
     fs.chmod(opts.keyout, '0600', function (err) {
```

Comment:
@jeffmendoza I thought about this fix. 
openssl rsa line rewrites the key removing the phassphrase. It is optional and should be executed to fix a strange behaviour happening on OSX, even if not works correctly it doesn't mean that the entire process should be stopped.
On the other hand fs.chmod line is mandatory.
Just to be more precise, I could change the openssl error message into a warning.